package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "todo-list-microverse-webpack",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --mode production"
+  },
```

Comment:
- [ ] Kindly make sure your webpack dev server is set up and running with no errors, your project should be able to be run using `npm start`